<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drone Taxi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="videoGraphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightSide">
      <item>
       <widget class="QProgressBar" name="batteryProgress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>Bat: %p%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="landingModeLabel">
        <property name="text">
         <string>LandingM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="modeLabel">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetPadLabel">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="panicButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Panic!!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="landButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Land</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
